functions/app/build.gradle.kts

```
@@ -36,7 +36,7 @@ dependencies {
     implementation("androidx.appcompat:appcompat:1.7.0")
 
     // Import the BoM for the Firebase platform
-    implementation(platform("com.google.firebase:firebase-bom:33.5.1"))
+    implementation(platform("com.google.firebase:firebase-bom:33.9.0"))
```

I'm not sure why the build keeps failing in GH Actions. Locally I'm able to build the functions module with no issues using the firebase-bom:33.9.0 dependency.

```suggestion
    implementation(platform("com.google.firebase:firebase-bom:33.5.1"))
```